### DISCUSSION TITLE
Windows 10 style taskbar does not show pinned icon in notification area

### DISCUSSION THREAD

**Discussion by riku22** [valinet/ExplorerPatcher#828] [Q&A]
Hello.

I'm not sure if this is the right place, but let me ask you a question.
After updating to Windows 11, I used the standard Windows 11 taskbar for a while.
At that time, I set some icons to always appear in the notification area in the Windows 11 taskbar settings screen.
However, when I switch to the Windows 10 style taskbar in the ExplorerPatcher settings, the icons that I set to always be displayed are hidden.
I can fix this by switching it on and off once from the Windows 11 settings screen, but is there anything I can do so that I don't have to do this?
Sorry for the confusing explanation.

Sincerely.

**Comment by valinet** [comment]
I don’t understand exactly, but the settings are separate because Microsoft edits the Windows 11 ones all the time to remove the standard icons (Internet, battery, sound etc) from there. Thus, the 10 taskbar having different settings, it doesn’t respect what you set for the 11 one.

Easiest to fix is to open the notification overflow, and drag the icons you want to see permanently to the system tray. That’s it, more reliable and it always works. You can sign out and then back in to see if the changes stick.

Thanks

**Reply by nbetcher** [reply]
FYI:

> I don’t understand exactly, but the settings are separate because Microsoft edits the Windows 11 ones all the time to remove the standard icons (Internet, battery, sound etc) from there. Thus, the 10 taskbar having different settings, it doesn’t respect what you set for the 11 one.

I tried editing the overflow (hidden) tray icons by using the Windows 10 Notification Tray icon Control Panel applet, but it didn't actually change anything, even if I restarted Explorer.

> Easiest to fix is to open the notification overflow, and drag the icons you want to see permanently to the system tray. That’s it, more reliable and it always works. You can sign out and then back in to see if the changes stick.

I tried this and it worked perfect! Crazy. Thanks

Update: On Explorer restart, the icon dragged to the tray (unhidden) not only didn't appear in the system tray (shown), but it's now also missing from the hidden menu too. :-(

Update 2: Only way to show icons the proper and persisting way is to show them all using the, "Always show all icons and notifications on the taskbar". Unfortunately, and also quite fascinatingly, this also does NOT restore the icon that went missing as part of dragging the icon described in my original post. It appears to now be permanently lost.

**Reply by JamesDBartlett3** [reply]
@nbetcher Have you found a fix for the missing icons after Explorer restart? I was about to try out @valinet's suggestion, but after seeing your update, I'm hesitating.

**Reply by nbetcher** [reply]
> @nbetcher Have you found a fix for the missing icons after Explorer restart? I was about to try out @valinet's suggestion, but after seeing your update, I'm hesitating. 

Not exactly... Eventually the icon came back after some time. What triggered it, I don't know.
